mariadb/scale-mariadb-server/scale-mariadb-server.sh

```diff
@@ -28,13 +28,20 @@ az monitor metrics list \
 --metric storage_used \
```

Comment:
Unable to comment, line L16:

__Recommended__: Use a generated password when creating a resource, and display it to the user at some point. See https://review.docs.microsoft.com/en-us/help/contribute/conventions-azure-cli?branch=master#randomize-passwords-for-new-resources

Reply:
Unable to comment, L12:

__Required__: The name of the database is required to be a unique identifier but isn't. See https://review.docs.microsoft.com/en-us/help/contribute/conventions-azure-cli?branch=master#avoid-naming-conflicts